src/main/java/com/nylas/Event.java

```diff
@@ -175,7 +177,7 @@ public void setRecurrence(Recurrence recurrence) {
 		}
 
 		List<Map<String, Object>> participantWritableFields = null;
-		if(!participants.isEmpty()) {
```

Comment:
if this was a null, could you just initialize the field with ArrayList<Participant>(). then its an empty list instead of checking for null. Same with the map, list and boolean fields in the object

Reply:
Thanks for your feedback, I'll implement it for lists now and going forward!